src/transports/sentry_transport_winhttp.c

```diff
@@ -286,3 +286,8 @@ sentry__transport_new_default(void)
 
     return transport;
 }
+
+void
+sentry__transport_dump_queue(sentry_transport_t *UNUSED(transport))
+{
+}
```

Comment:
Is this intentionally empty?

Reply:
well, kind of. Windows doesn’t have any inproc backend, so this is basically useless. I can implement it, its basically the same as for the curl backend, so there are no surprises once windows will get an inproc backend.

Reply:
👍 that would be great. Otherwise it might be too easy to miss this.